torch/csrc/distributed/c10d/socket.cpp

```diff
@@ -870,8 +876,8 @@ bool SocketConnectOp::tryConnect(int family) {
           retry_attempt = 0;
         }
```

Comment:
This piece of code is now inaccurate with the `retry_attempt == 30`. 
It'll be more than 30 seconds now as we are using Exponential backoff here right?

Reply:
done